nntrainer/layers/layer_devel.h

```diff
@@ -174,6 +171,20 @@ class Layer {
 using CreateLayerFunc = nntrainer::Layer *(*)();
 using DestroyLayerFunc = void (*)(nntrainer::Layer *);
 
+/**
```

Comment:
this is fine for now, but later you can consider putting `LayerPluggable` stuff together, and moving `createLayer` out from the middle of it's related declarations and definitions.